docs/Scaling compute/Polly CLI.md

```diff
@@ -299,16 +322,36 @@ A prompt to enter job id will appear which when kept blank gets all the job stat
 
 ![All Job Statuses in a Workspace](../img/PollyCLI/6.png "All Job Statuses in a Workspace") <center>**Figure 6.** All Job Statuses in a Workspace</center>
 
+*   If you are within a Polly environment while accessing Polly CLI (like notebook or CLI job), workspace id can be auto-detected by passing --yes or -y in the command instead of --workspace-id parameter.
+
 **Get job logs**
 
+To view the logs of any job, use the following command.
+
 <pre><code>polly jobs logs --workspace-id <workspace id> --job-id <job id></code></pre>
 
 This will give the logs for the job. In case the job is still running, it will give the logs generated till that instant.
 
 ![Job Logs](../img/PollyCLI/7.png "Job Logs") <center>**Figure 7.** Job Logs</center>
 
+**Note** - If you are within a Polly environment while accessing Polly CLI (like notebook or CLI job), workspace id can be auto-detected by passing --yes or -y in the command instead of --workspace-id parameter.
```

Comment:
Follow the same format for Note as in the rest of the documentation. The text should be as a bullet point in the next line.